packages/features/ee/round-robin/roundRobinManualReassignment.ts

```diff
@@ -266,11 +300,11 @@ export const roundRobinManualReassignment = async ({
     cancelledEvt,
     [
       {
-        ...originalOrganizer,
-        name: originalOrganizer.name || "",
-        username: originalOrganizer.username || "",
-        timeFormat: getTimeFormatStringFromUserTimeFormat(originalOrganizer.timeFormat),
-        language: { translate: originalOrganizerT, locale: originalOrganizer.locale || "en" },
+        ...previousRRHost,
```

Comment:
The `previousRRHost` needs to get the cancellation emails. The orginalOriganizer could be a fixed host